src/dsp_tools/models/exceptions.py

```diff
@@ -128,6 +82,17 @@ class UserError(BaseError):
     """
 
 
+class PermanentConnectionError(BaseError):
+    """
+    This error is raised when all attempts to reconnect to DSP have failed.
+
+    Attributes:
+        message: A message that describes the error
+    """
+
+    message: str
+
+
 class XmlUploadError(Exception):
```

Comment:
I know this is not the scope, but while you're at it could you subclass this of a BaseError or is there are reason this is an Exception?